Mathlib/NumberTheory/MulChar/Duality.lean

```diff
@@ -0,0 +1,75 @@
+/-
+Copyright (c) 2024 Michael Stoll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Stoll
+-/
+import Mathlib.GroupTheory.FiniteAbelian.Duality
+import Mathlib.NumberTheory.MulChar.Basic
+
+/-!
+# Duality for multiplicative characters
+
+Let `M` be a finite commutative monoid and `R` a ring that has enough `n`th roots of unity,
+where `n` is the exponent of `M`. Then the main results of this file are as follows.
+
+* `MulChar.exists_apply_ne_one_of_hasEnoughRootsOfUnity`: multiplicative characters
+  `M → R` separate elements of `M`.
```

Comment:
This comment suggests that for each two `a b : M` there is a character that sends `a` and `b` to different images (if `a` and `b` differ). But that's not true for non-units, right? Is this standard terminology?

Reply:
You're right; I'll change to "separate elements of `Mˣ`".